File: model/Card.java
package model;

import model.common.PocketObject;

public class Card extends PocketObject {
  private final CardType type;

  public Card(int id, CardType type) {
    super(id);
    this.type = type;
  }

  public CardType getType() {
    return type;
  }

  @Override
  public String toString() {
    return "Card{" +
        "type=" + type +
        '}';
  }
}

File: model/Card.py
from model.common.PocketObject import PocketObject


class Card(PocketObject):
    def __init__(self, id, card_type):
        super().__init__(id)
        self._type = card_type

    def get_type(self):
        return self._type

    def __str__(self):
        return f"Card{{type={self._type}}}"
